packages/features/webhooks/lib/factory/BookingPayloadBuilder.ts

```diff
@@ -0,0 +1,206 @@
+import { getUTCOffsetByTimezone } from "@calcom/lib/dayjs";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+import type { EventTypeInfo, BookingWebhookEventDTO } from "../dto/types";
+import type { WebhookPayload } from "./types";
+
+type BookingExtraDataMap = {
+  [WebhookTriggerEvents.BOOKING_CREATED]: undefined;
```

Comment:
nit: Wouldn't it make more sense to set it as _null_.  _undefined_ means no value is defined and null means it has no value.

Reply:
Makes sense!